tox.ini

```diff
@@ -72,8 +72,9 @@ pass_env =
     CI_PACKED_CHARMS
```

Comment:
fun fact, if you want to work with juju `2.9` with the pyyaml conflicts, you can use `2.9.44.0`